library/ssl_tls.c

```diff
@@ -3933,6 +3933,22 @@ void mbedtls_ssl_conf_sig_hashes( mbedtls_ssl_config *conf,
 {
     conf->sig_hashes = hashes;
 }
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+/**
```

Comment:
I don't think duplicating the public documentation here is a good idea. Let's just have a brief, non-doxygen comment, such as `/* Configure allowed signature algorithms for use in TLS 1.3 */` and leave the rest in the header file.